cypress/integration/login.js

```diff
@@ -20,5 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('[data-test="timesheets-current-day"]').should('contain', 15)
```

Comment:
This test will fail tomorrow, maybe we can use something better? e.g. a hint https://www.w3schools.com/jsref/jsref_getday.asp

Reply:
thanks

Reply:
https://www.w3schools.com/jsref/jsref_getdate.asp